backends/v3/src/radix.rs

```diff
@@ -1,10 +1,17 @@
 use crate::block_allocator::{Allocator, BlockAllocation};
 use slotmap::{DefaultKey, SlotMap};
+use std::hash::{Hash, Hasher};
 use std::{
     collections::{BTreeSet, HashMap},
     sync::Arc,
 };
 
+fn hash_(slice: &[u32]) -> u64 {
```

Comment:
For slice length 1, we can return `slice[0] as u64`. Maybe also an assertion that the slice length is at least 1?

Reply:
Done